cli/reporting.go

```diff
@@ -0,0 +1,215 @@
+package cli
+
+import (
+	"context"
+	"time"
+
+	"github.com/cheynewallace/tabby"
+	"github.com/mongodb/amboy/reporting"
+	"github.com/mongodb/amboy/rest"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+func reports(opts *ServiceOptions) cli.Command {
+	return cli.Command{
+		Name: "report",
+		Subcommands: []cli.Command{
+			reportingJobStatus(opts),
+			reportingRecentTiming(opts),
+			reportingJobIDs(opts),
+			reportingRecentErrors(opts),
+		},
+	}
+}
+
+func reportingJobStatus(opts *ServiceOptions) cli.Command {
+	return cli.Command{
+		Name: "status",
+		Flags: opts.reportingFlags(
+			cli.StringFlag{
+				Name:  "filter",
+				Value: "in-progress",
+				Usage: "specify the process filter, can be 'in-progress' (default), 'pending', or 'stale'",
```

Comment:
The "default" is superfluous since the cli should already print the default when Value is set.

Reply:
done.